thoth/storages/graph/postgres.py

```diff
@@ -4761,9 +4761,13 @@ def _create_python_package_version(
             os_version=os_version,
             python_version=python_version,
             entity_id=entity.id,
-            python_package_metadata_id=python_package_metadata_id,
         )
 
+        # including this value in "get_or_create" will cause errors because it is not part of a unique entry
+        if python_package_metadata_id:
+            python_package_version.python_package_metadata_id = python_package_metadata_id
+            session.commit()
```

Comment:
this commit cannot be done here because this method is used in other methods so there are still opreations to be done later. for example it is used during solver sync method (in particular to create the solved record in the Solved table: https://github.com/thoth-station/storages/blob/feec955fb80f84006659662bad7916520de7907f/thoth/storages/graph/postgres.py#L6179).

Testing you get the following error:

```
Traceback (most recent call last):
  File "mix_test.py", line 42, in <module>
    output = sync_solver_documents(
  File "/home/fmurdaca/work/aicoe/storages/thoth/storages/sync.py", line 131, in sync_solver_documents
    graph.sync_solver_result(document, force=force)
  File "/home/fmurdaca/work/aicoe/storages/thoth/storages/graph/postgres.py", line 6264, in sync_solver_result
    solved, _ = Solved.get_or_create(
  File "/home/fmurdaca/.local/share/virtualenvs/storages-kD6ztVxe/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 606, in __exit__
    self._assert_active(deactive_ok=True, prepared_ok=True)
  File "/home/fmurdaca/.local/share/virtualenvs/storages-kD6ztVxe/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 305, in _assert_active
    raise sa_exc.ResourceClosedError(closed_msg)
sqlalchemy.exc.ResourceClosedError: This transaction is closed
```